packages/main/src/Tree.ts

```diff
@@ -0,0 +1,515 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import slot from "@ui5/webcomponents-base/dist/decorators/slot.js";
+import event from "@ui5/webcomponents-base/dist/decorators/event.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { getEffectiveAriaLabelText } from "@ui5/webcomponents-base/dist/util/AriaLabelHelper.js";
+import TreeItem from "./TreeItem.js";
+import TreeItemCustom from "./TreeItemCustom.js";
+import TreeList from "./TreeList.js";
+import ListMode from "./types/ListMode.js";
+import type {
+	TreeItemBaseToggleEventDetail,
+	TreeItemBaseStepInEventDetail,
+	TreeItemBaseStepOutEventDetail,
+} from "./TreeItemBase.js";
+import type {
+	ClickEventDetail as ListItemClickEventDetail,
+	DeleteEventDetail as ListItemDeleteEventDetail,
+	SelectionChangeEventDetail as ListSelectionChangeEventDetail,
+} from "./List.js";
+
+// Template
+import TreeTemplate from "./generated/templates/TreeTemplate.lit.js";
+
+// Styles
+import TreeCss from "./generated/themes/Tree.css.js";
+import TreeItemBase from "./TreeItemBase.js";
```

Comment:
TreeItemBase is used only as a type. Add type and move it to other imports from ./